spec/requests/partners/dashboard_requests_spec.rb

```diff
@@ -17,8 +17,11 @@
 
   describe "GET #index" do
     it "displays requests that are pending" do
-      create(:request, :pending, partner: partner, request_items: [{item_id: item1.id, quantity: "16"}])
-      create(:request, :pending, partner: partner, request_items: [{item_id: item2.id, quantity: "16"}])
+      request1 = create(:request, :pending, partner: partner, request_items: [])
+      create(:item_request, request: request1, quantity: 16, item: item1)
+
+      request2 = create(:request, :pending, partner: partner, request_items: [])
+      create(:item_request, request: request2, quantity: 16, item: item2)
```

Comment:
What are these changes about?

Reply:
This is because we changed the view from using `request.request_items` to `request.item_requests` in the view itself. This spec previously ONLY set up the `request_items`, which is like a weird cached copy of the real `item_requests`. Once we switched the view we had to update the test to make the real records.